src/app/zap-templates/zcl/data-model/chip/fan-control-cluster.xml

```diff
@@ -48,20 +48,14 @@ limitations under the License.
     <item name="Off/On" value="0x05"/>
   </enum>
 
-  <bitmap name="RockSupportMask" type="BITMAP8">
+  <bitmap name="RockBitmap" type="BITMAP8">
     <cluster code="0x0202" />
     <field name="RockLeftRight" mask="0x01" />
     <field name="RockUpDown" mask="0x02" />
     <field name="RockRound" mask="0x04" />
   </bitmap>
 
-  <bitmap name="WindSupportMask" type="BITMAP8">
-    <cluster code="0x0202" />
-    <field name="Sleep Wind" mask="0x01" />
-    <field name="Natural Wind" mask="0x02" />
-  </bitmap>
-
-  <bitmap name="WindSettingMask" type="BITMAP8">
+  <bitmap name="WindBitmap" type="BITMAP8">
     <cluster code="0x0202" />
     <field name="Sleep Wind" mask="0x01" />
     <field name="Natural Wind" mask="0x02" />
```

Comment:
DirectionEnum needs to become StepDirectionEnum, no?  I know it was not part of the editorial PR but some other PR (which didn't file an SDK issue to track), but still.